hack/config-default.sh

```diff
@@ -4,7 +4,7 @@ docker_tag=${DOCKER_TAG:-latest}
 docker_tag_alt=${DOCKER_TAG_ALT}
 image_prefix=${IMAGE_PREFIX}
 image_prefix_alt=${IMAGE_PREFIX_ALT}
-namespace=kubevirt
+namespace=${KUBEVIRT_INSTALLED_NAMESPACE:-kubevirt}
 deploy_testing_infra=${DEPLOY_TESTING_INFRA:-false}
 csv_namespace=placeholder
 cdi_namespace=cdi
```

Comment:
should we also customize `CDI_NAMESPACE`?

Reply:
@danielBelenky Currently there is no need to modify CDI installation namespace here because the consume of CDI generated manifests is done from the CDI project. Probably should be added there.